sklearn/model_selection/tests/test_split.py

```diff
@@ -199,6 +199,30 @@ def test_cross_validator_with_default_params():
                          lpo.get_n_splits, None, y, groups)
 
 
+def test_2d_y():
+    # smoke test for 2d y and multi-label
+    n_samples = 30
+    rng = np.random.RandomState(1)
+    X = rng.randint(0, 3, size=(n_samples, 2))
+    y = rng.randint(0, 3, size=(n_samples,))
+    y_2d = y.reshape(-1, 1)
+    y_multilabel = rng.randint(0, 2, size=(n_samples, 3))
+    groups = rng.randint(0, 3, size=(n_samples,))
+    splitters = [LeaveOneOut(), LeavePOut(p=2), KFold(), StratifiedKFold(),
```

Comment:
should this be a global or generated by a global function?

Reply:
I was considering it. A global variable seems consistent with other lists of classes, but I didn't do that given that it's only used once so far.

Reply:
pytest.mark.parametrize?